operations/helm/charts/mimir-distributed/templates/_helpers.tpl

```diff
@@ -74,6 +74,18 @@ Create the name of the service account
 {{- end -}}
 {{- end -}}
 
+{{/*
+Create the name of the ruler service account
+*/}}
+{{- define "mimir.ruler.serviceAccountName" -}}
+{{- if .Values.ruler.serviceAccount.create -}}
+{{- $sa := default (include "mimir.fullname" .) .Values.serviceAccount.name }}
+{{- printf "%s-%s" $sa "ruler" }}
```

Comment:
so if I provide

```yaml
ruler:
  serviceAccont: 
    name: ruler-sa
```

the service account will have the name `ruler-sa-ruler`. Is the `-ruler` suffix necessary here?